src/code/z_actor.c

```diff
@@ -2707,19 +2708,88 @@ void Actor_DrawLensActors(PlayState* play, s32 numInvisibleActors, Actor** invis
     CLOSE_DISPS(gfxCtx, "../z_actor.c", 6284);
 }
 
-s32 func_800314B0(PlayState* play, Actor* actor) {
-    return func_800314D4(play, actor, &actor->projectedPos, actor->projectedW);
+/**
+ * Checks if an actor should be culled or not, by seeing if it is contained within it's own culling volume.
+ * For more details on the culling test, see `Actor_CullingVolumeTest`.
+ *
+ * "Culling" in this context refers to the removal of something for the sake of improving performance.
+ * For actors, being culled means that their Update and Draw processes are halted.
+ * While halted, the Actor's update state is frozen and they will not draw, making them invisible.
+ *
+ * Actors that are in bounds of it's culling volume may update and draw, while actors that are
+ * out of bounds of it's culling volume may be excluded from updating and drawing until they are within bounds.
```

Comment:
```suggestion
 * Actors that are within bounds of their culling volume may update and draw, while actors that are
 * out of bounds of their culling volume may be excluded from updating and drawing until they are within bounds.
```

Reply:
I think "are in bounds" is fine, but if you want to use "within" it should be "within the bounds"